docs/products/opensearch/howto/opensearch-log-integration.rst

```diff
@@ -1,5 +1,5 @@
-Manage OpenSearch log integration
-=================================
+Manage OpenSearch logs integration
```

Comment:
I think it should be kept as it is as "log integration". When I search for titles with similar names across other websites, it is common to keep "log" single.

@TibsAtWork , any opinion on this?

Reply:
I can leave as it is

Reply:
Sorry, I think "log integration" singular as well